src/Uno.UI/Helpers/WinUI/ResourceAccessor.cs

```diff
@@ -61,18 +61,153 @@ public static string GetLocalizedStringResource(string resourceName)
 
 		public static LoadedImageSurface GetImageSurface(string assetName, Size imageSize)
 		{
-			Uri getImageUri() {
+			Uri getImageUri()
+			{
 				if (SharedHelpers.IsInFrameworkPackage())
 				{
 					return new Uri("ms-resource://" + MUXCONTROLS_PACKAGE_NAME + "/Files/Microsoft.UI.Xaml/Assets/" + assetName + ".png");
 				}
 				else
 				{
-					return new Uri("ms-resource:///Files/Microsoft.UI.Xaml/Assets/"+assetName  + ".png");
+					return new Uri("ms-resource:///Files/Microsoft.UI.Xaml/Assets/" + assetName + ".png");
 				}
 			}
 
 			return LoadedImageSurface.StartLoadFromUri(getImageUri(), imageSize);
 		}
+
+		// Header file
+
+		internal const string SR_BasicRatingString = "BasicRatingString";
```

Comment:
For the ColorPicker I've also copied these resource key variables into Uno. However, I split them apart into a partial class: ResourceAccessor.ResourceKeys.cs. I think this might be cleaner as we add lots more keys over time. Each section should also have a comment indicating what control the keys are for I think.